src/exceptions/Exception.class.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Tripod\Exceptions;
+
+/**
+ * Class Exception
+ * @package Tripod
+ */
+class Exception extends \Exception {}
```

Comment:
What is this adding over \Exception? The other exceptions in this namespace give something more specific in their name. Should this one?